.circleci/config.yml

```diff
@@ -309,3 +320,9 @@ workflows:
           filters:
             branches:
               only: master
+      - aws-sync:
+          requires:
+            - build-linux
```

Comment:
this would require the darwin and freebsd builds as well, no?

Reply:
Ah good catch! I've made this PR only implement syncing for docs now though.